core/interfaces/i_variable_map.ts

```diff
@@ -60,13 +62,4 @@ export interface IVariableMap<T extends IVariableModel, U extends State> {
 
   /* Removes all variables from this variable map. */
   clear(): void;
-
-  /* Returns an object representing the serialized state of the variable. */
-  saveVariable(variable: T): U;
```

Comment:
Is this deletion a breaking change?

Reply:
No, this interface was added a couple days ago in the V12 branch, it's not yet been released.